doc/Clusters_from_Scratch/en-US/Ch-Active-Active.txt

```diff
@@ -1,755 +1,364 @@
-= Conversion to Active/Active =
-
-== Requirements ==
+= Convert Cluster to Active/Active =
 
 The primary requirement for an Active/Active cluster is that the data
 required for your services is available, simultaneously, on both
-machines. Pacemaker makes no requirement on how this is achieved, you
-could use a SAN if you had one available, however since DRBD supports
-multiple Primaries, we can also use that.
+machines. Pacemaker makes no requirement on how this is achieved; you
+could use a SAN if you had one available, but since DRBD supports
+multiple Primaries, we can continue to use it here.
+
+== Install Cluster Filesystem Software ==
 
 The only hitch is that we need to use a cluster-aware filesystem. The
 one we used earlier with DRBD, ext4, is not one of those. Both OCFS2
-and GFS2 are supported, however here we will use GFS2 which comes with
-Fedora 17.
-
-=== Installing the required Software ===
+and GFS2 are supported; here, we will use GFS2.
 
-[source,C]
+On both nodes, install the GFS2 command-line utilities and the
+Distributed Lock Manager (DLM) required by cluster filesystems:
+----
 # yum install -y gfs2-utils dlm kernel-modules-extra
```

Comment:
% rpm -q --whatrequires kernel-modules-extra
dlm-4.0.1-4.fc21.x86_64

will remove explicit install in next round